src/extract/ourairports.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import geopandas as gpd
+import pandas as pd
+
+next_link = "https://ourairportapi.com/airports-in/Yemen"
+iso3 = 'YEM'
+
+# Loop through pages and pull data from the API
+session = requests.Session()
+
+print(next_link)
+request = session.get(url=next_link)
+json = request.json()
+
+# Loop through flows and append data to dataframe
+airports = json["results"]
+print(airports)
+airport_list = []
+for airport in airports:
+    airport_list.append(airport)
+
+airport_df = pd.DataFrame(airport_list)
+print(airport_df)
+airport_df = airport_df.drop(columns=['keywords'])
+
+airport_pt = gpd.GeoDataFrame(airport_df,
+    geometry=gpd.points_from_xy(airport_df.lon, airport_df.lat))
+
+# Set CRS
+airport_pt.crs = {'init': 'epsg:4326'}
```

Comment:
The projection is declared as 4326 as a string - that's fine but is this always true for ourairports?